ziti/cmd/ascode/exporter/exporter.go

```diff
@@ -38,24 +38,29 @@ import (
 var log = pfxlog.Logger()
 
 type Exporter struct {
-	loginOpts        edge.LoginOptions
-	client           *rest_management_api_client.ZitiEdgeManagement
-	ofJson           bool
-	ofYaml           bool
+	Out              io.Writer
+	Err              io.Writer
+	LoginOpts        edge.LoginOptions
+	OfJson           bool
```

Comment:
OfJson and OfYaml don't make sense to me. How about OutputType and take json or yaml with yaml being the default and eliminate one of the flags?